packages/graphql/src/schemaTypes/objectTypes/gql-LocalSettings.ts

```diff
@@ -36,6 +36,15 @@ export const LocalSettingsPreferences = objectType({
     t.boolean('debugSlideshowComplete')
     t.boolean('desktopNotificationsEnabled')
     t.dateTime('dismissNotificationBannerUntil')
+    t.boolean('notifyWhenRunStarts')
+    t.boolean('notifyWhenRunStartsFailing')
+    t.json('notifyWhenRunCompletes', {
```

Comment:
I tried to use a GraphQL List type with an enum for this, but I couldn't get the initial values to load properly unless I did this.